GUI/QtQuick/qml/Send/SimpleDetails.qml

```diff
@@ -125,14 +126,48 @@ ColumnLayout  {
 
         //aliases
         title_text: qsTr("Amount")
-        input_text: "0"
-
-        input_validator: DoubleValidator{
-            bottom: 0
-            top: (from_wallet_combo.currentIndex >= 0) ?
-                     parseFloat(getWalletData(from_wallet_combo.currentIndex, WalletBalance.TotalRole)) : 0
-            notation :DoubleValidator.StandardNotation
-            locale: "en_US"
+
+//                input_validator: DoubleValidator{
+//                    bottom: 0
+//                    top: (from_wallet_combo.currentIndex >= 0) ?
+//                             parseFloat(getWalletData(from_wallet_combo.currentIndex, WalletBalance.TotalRole)) : 0
+//                    notation :DoubleValidator.StandardNotation
+//                }
+
+        //visual studio crashes when there is input_validator
```

Comment:
Not VS, but the program itself, I believe

Reply:
Yes application when it was launching in Visual studio. (in qt creator all was ok)